services/director-v2/src/simcore_service_director_v2/api/routes/dynamic_services.py

```diff
@@ -342,6 +344,15 @@ async def update_projects_networks(
     )
 
 
+def _is_service_inactive(activity_info: ActivityInfoOrNone, threshold: float) -> bool:
+    if activity_info is None:
+        # services which do not support inactivity are treated as being inactive
+        return True
+
+    is_inactive: bool = activity_info.seconds_inactive >= threshold
```

Comment:
```python
return activity_info.seconds_inactive >= threshold
```
or if the mypy annoys you, just cast it as
```python
return bool(activity_info.seconds_inactive >= threshold)
```

Reply:
I avoid casting to bool since the value is already a bool value.
This makes it expliclt, tha the isue is the typing/tooling